swagger.json

```diff
@@ -3264,6 +3264,11 @@
                                 "type": "string",
                                 "description": "Name of location at which the log was recorded.",
                                 "example": "McLean Site A"
+                            },
+                            "remark":{
+                                "type": "string",
+                                "description": "Remarks associated with the log entry.",
```

Comment:
Could we use "Remark" instead of "Remarks", to keep the plurality consistent with the key name?